sci-geosciences/gpxlab/gpxlab-0.5.0.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Program to show and manipulate GPS tracks"
+DESCRIPTION="GPXLab is an application to display and manage GPS tracks \
+previously recorded with a GPS tracker."
+HOMEPAGE="https://bourgeoislab.wordpress.com/gpxlab/"
+COPYRIGHT="2018 Frédéric Bourgeois"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/BourgeoisLab/GPXLab/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="430672eaf0c76e1f7f43cf2c117e63d818ac7e444acceefe6a0ef5a2f7b3d7f4"
+SOURCE_DIR="GPXLab-$portVersion"
```

Comment:
Thanks @sikmir for your PR!

It would be nice to define a `SOURCE_FILENAME`:
```diff
 CHECKSUM_SHA256="430672eaf0c76e1f7f43cf2c117e63d818ac7e444acceefe6a0ef5a2f7b3d7f4"
+SOURCE_FILENAME="GPXLab-$portVersion.tar.gz"
 SOURCE_DIR="GPXLab-$portVersion"
```
Ref: [HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name)

Reply:
Oops, too late... Never mind.
Next time I'll add a "wip-no-merge" :smile: